pipes/WDL/workflows/scaffold_and_refine_multitaxa.wdl

```diff
@@ -0,0 +1,112 @@
+version 1.0
+
+import "../tasks/tasks_assembly.wdl" as assembly
+import "../tasks/tasks_ncbi.wdl" as ncbi
+import "../tasks/tasks_utils.wdl" as utils
+import "assemble_refbased.wdl" as assemble_refbased
+
+workflow scaffold_and_refine_multitaxa {
+    meta {
+        description: "Scaffold de novo contigs against a set of possible references and subsequently polish with reads."
+        author: "Broad Viral Genomics"
+        email:  "viral-ngs@broadinstitute.org"
+        allowNestedInputs: true
+    }
+
+    input {
+        String  sample_id
+        File    reads_unmapped_bam
+
+        Array[Pair[Int,Array[String]+]] taxid_to_ref_accessions = [
```

Comment:
What about HAdV?
There are [quite a few](https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=10509&host=human), but if we do want to include them:
```
            # HAdV reference genomes, via https://www.ncbi.nlm.nih.gov/genomes/GenomesGroup.cgi?taxid=10509&host=human
            (129875,  ["NC_001460.1"]),  # Human mastadenovirus A strain:Huie; serotype:Human adenovirus 12; culture-collection:ATCC:VR-863
            (108098,  ["NC_011203.1"]),  # Human adenovirus B1
            (108098,  ["NC_011202.1"]),  # Human adenovirus B2
            (129951,  ["NC_001405.1"]),  # Human mastadenovirus C serotype:Human adenovirus 2
            (130310,  ["NC_010956.1"]),  # Human mastadenovirus D strain:Hicks; NIAID V-209-003-014; serotype:Human adenovirus 9
            (130308,  ["NC_003266.2"]),  # Human mastadenovirus E strain:vaccine (CL 68578); serotype:human adenovirus 4
            (130309,  ["NC_001454.1"]),  # Human mastadenovirus F strain:Dugan; serotype:Human adenovirus 40
            (310540,  ["NC_006879.1"]),  # Simian adenovirus 1 strain:ATCC VR-195
            (1123958, ["NC_017825.1"]),  # Chimpanzee adenovirus Y25
```